src/types/route-requests.ts

```diff
@@ -88,6 +88,7 @@ export type AccessCodeCreateMultipleRequest =
 export interface AccessCodeUpdateBaseRequest {
   device_id?: string
   access_code_id: string
+  type?: "ongoing" | "time_bound"
```

Comment:
Should this be added as a literal on each request type or just here as an optional union?